### PR TITLE
Custom topology test integration

### PR DESCRIPTION
**Pull request by b00ris**
closes https://github.com/clearmatics/autonity/issues/355
closes https://github.com/clearmatics/autonity/issues/331

### CODE REVIEW

File: common/graph/grammer_test.go [hunk 1]
```diff
@@ -67,6 +68,7 @@ func TestGraphLexerGetEdges(t *testing.T) {
 	for i, name := range nodeNames {
 		graph.SetNodeName(name, i)
 	}
+	spew.Dump(nodeNames)
```
Comment (JekaMas) [inline common/graph/grammer_test.go]: Is it a kind of debug?

File: consensus/test/autonity_contract_test.go [hunk 2]
```diff
@@ -0,0 +1,335 @@
+package test
+
+/*
```
Comment (JekaMas) [inline consensus/test/autonity_contract_test.go]: Is it a debug or it should be restored?

File: consensus/test/topology_test.go [hunk 3]
```diff
@@ -0,0 +1,114 @@
...
+		})
+	}
+}
+
+func TestTendermintBusSuccess(t *testing.T) {
+	if testing.Short() {
+		t.Skip("skipping test in short mode")
+	}
+
+	topologyStr := `graph TB
```
Comment (JekaMas) [inline consensus/test/topology_test.go]: Please, comment in the PR that this topology descriptions will be moved to files in a new issue.

File: consensus/test/autonity_contract_test.go (hunk 2 above)
Comment (b00ris) [reply consensus/test/autonity_contract_test.go]: Sure. I forgot to restore

File: consensus/test/topology_test.go (hunk 3 above)
Comment (b00ris) [reply consensus/test/topology_test.go]: Yes. I was going to make it as a separate issue.

File: consensus/test/autonity_contract_test.go (hunk 2 above)
Comment (b00ris) [reply consensus/test/autonity_contract_test.go]: fixed

File: consensus/test/topology_test.go [hunk 4]
```diff
@@ -0,0 +1,114 @@
...
+    PF---VA
+    PF---VB
+    PF---VC
+    PF---VD
+    PF-->VE`
```
Comment (yazzaoui) [inline consensus/test/topology_test.go]: Participants don't gossip consensus messages, how could this work?

**Pull request merged**